site/en/docs/web-platform/conditional-focus/index.md

```diff
@@ -0,0 +1,185 @@
+More precisely:
+- The `getDisplayMedia()` returned promise resolves on a microtask. Calling `setFocusBehavior()` after that microtask completes throws an error.
+- Calling `setFocusBehavior()` more than a second after capture starts is no-op.
+
+That is, both of the following snippets will fail:
+
+```js
+// Prompt the user to share their screen.
+const stream = await navigator.mediaDevices.getDisplayMedia({ controller });
+
+// Too late, because it follows the completion of the task
+// on which the getDisplayMedia() promise resolved.
+// This will throw.
+setTimeout(() => {
+  controller.setFocusBehavior("focus-captured-surface");
+});
+```
+
+```js
+// Prompt the user to share their screen.
+const stream = await navigator.mediaDevices.getDisplayMedia({ controller });
+
+const start = new Date();
+while (new Date() - start <= 1000) {
+  // Idle for ≈1s.
+}
+
+// Because too much time has elapsed, the browser will have
+// already decided whether to focus.
+// This fails silently.
+controller.setFocusBehavior("focus-captured-surface");
+```
+
+Calling `setFocusBehavior()` also throws in the following cases:
+
+- the video track of the stream returned by `getDisplayMedia()` is not ["live"].
+- after the `getDisplayMedia()` returned promise resolves, if the user shared a screen (not a tab or a window).
+
+## Sample  {: #sample }
+
+You can play with Conditional Focus by running the [demo] on Glitch. Be sure to [check out the source code].
+
+{% Glitch { id: 'conditional-focus', path: 'index.js', previewSize: 0 } %}
+
+## Feature detection {: #feature-detection }
+
+To check if `CaptureController.setFocusBehavior()` is supported, use:
+
+```js
+if (
+  "CaptureController" in window &&
+  "setFocusBehavior" in CaptureController.prototype
+) {
+  // CaptureController.setFocusBehavior() is supported.
+}
+```
+
+## Feedback {: #feedback }
+
+The Chrome team and the web standards community want to hear about your experiences with Conditional Focus.
+
+### Tell us about the design
+
+Is there something about Conditional Focus that doesn't work as you expected? Or are there missing methods or properties that you need to implement your idea? Have a question or comment on the security model?
+
+* File a spec issue on the [GitHub repo][issues], or add your thoughts to an existing issue.
+
+### Problem with the implementation?
+
+Did you find a bug with Chrome's implementation? Or is the implementation different from the spec?
+
+* File a bug at <https://new.crbug.com>. Be sure to include as much detail as you can, and simple instructions for reproducing. [Glitch](https://glitch.com) works well for sharing code.
+
+### Show support
+
+Are you planning to use Conditional Focus? Your public support helps the Chrome team prioritize features and shows other browser vendors how critical it is to support them.
+
+Send a tweet to [@ChromiumDev] and let us know where and how you are using it.
+
+## Helpful links {: #links }
+
+- [Specification][spec]
+- [explainer]
```

Comment:
1) Capitalize E
2) Shall we add it in the other doc as well, or will it just link to this doc?

Reply:
1. Good catch!
2. Nope, we're good.